doc/reference/reference_lua/box_iproto/override.rst

```diff
@@ -0,0 +1,78 @@
+.. _reference_lua-box_iproto_override:
+
+box.iproto.override()
+=====================
+
+..  module:: box.iproto
+
+..  function:: override(request_type, handler)
+
+    Since version :doc:`2.11.0 </release/2.11.0>`.
+    Set a new IPROTO request handler callback for the given request type.
+
+    :param number request_type: a request type code. Possible values:
+
+                                *   a type code from :ref:`box.iproto.type <reference_lua-box_iproto_type>` (except
+                                    ``box.iproto.type.UNKNOWN``) -- override the existing request type.
+
+                                *   ``box.iproto.type.UNKNOWN`` -- override an unknown request type.
+
+    :param function handler: IPROTO request handler.
+                             The signature of a handler function: ``function(sid, header, body)``, where
+
+                             *  ``sid`` (number): current IPROTO session identifier (see :ref:`box.session.id() <box_session-id>`)
+                             *  ``header`` (userdata): a request header encoded as :ref:`msgpack_object <msgpack-object-methods>`
+                             *  ``body`` (userdata): a request body encoded as :ref:`msgpack_object <msgpack-object-methods>`
+
+                             Returns ``true`` on success, otherwise ``false``. In case of failure, there is a fallback
+                             to the system handler.
```

Comment:
I would mention, that one can indicate an error by throwing an exception. In this sense, retuning `false` means falling back to the system (default) handler, and this is not always a failure, it's just one of the possible options.